contracts/.gitignore

```diff
@@ -0,0 +1,12 @@
+node_modules
```

Comment:
@jdhyun09 Could you move those `gitignore` folders/files to the main gitignore file in the root folder?